### PR TITLE
Pluging in new Simulation OutputMapings and Charts

### PR DESCRIPTION
**Pull request by georgeDaskalakis**

### CODE REVIEW

**Comment by georgeDaskalakis** [comment]
@msevestre converting this to a draft, since moving code from PK-Sim to Core will also affect this PR.

**Comment by georgeDaskalakis** [comment]
@msevestre this is also ready with the latest Core dlls.

File: src/MoBi.Core/Serialization/Xml/Serializer/MoBiSimulationXmlSerializer.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ public MoBiSimulationXmlSerializer() : base(SerializationConstants.MoBiSimulatio
       public override void PerformMapping()
       {
          base.PerformMapping();
-         Map(x => x.Results);
+         Map(x => x.ResultsDataRepository);
```
Comment (msevestre) [inline src/MoBi.Core/Serialization/Xml/Serializer/MoBiSimulationXmlSerializer.cs:19]: This is not good here. You are breaking serialization

File: src/MoBi.Presentation/Presenter/ChartPresenter.cs [hunk 2]
```diff
@@ -75,6 +86,7 @@ public abstract class ChartPresenter : ChartPresenter<CurveChart, IChartView, IC
 
          _chartTemplatingTask = chartTemplatingTask;
          _dataRepositoryCache = new Cache<DataRepository, IMoBiSimulation>(onMissingKey: x => null);
+         _OutputMappingMatchingTask = OutputMappingMatchingTask;
```
Comment (msevestre) [inline src/MoBi.Presentation/Presenter/ChartPresenter.cs:89]: Casing is off here. Follow coding standard here

**Review by msevestre — CHANGES_REQUESTED** [review]
The serialization isa problem
You need to name the mapping explicitly to be like the old one

File: src/MoBi.Presentation/Presenter/ChartPresenter.cs (hunk 2 above)
Comment (georgeDaskalakis) [reply src/MoBi.Presentation/Presenter/ChartPresenter.cs:89]: done

File: src/MoBi.Core/Serialization/Xml/Serializer/MoBiSimulationXmlSerializer.cs (hunk 1 above)
Comment (georgeDaskalakis) [reply src/MoBi.Core/Serialization/Xml/Serializer/MoBiSimulationXmlSerializer.cs:19]: Not sure how you mean. Let's briefly discuss this today.

**Comment by georgeDaskalakis** [comment]
@msevestre is there something more to be changed for this PR? I would say this is ready for merging

**Pull request merged**